app/src/main/kotlin/com/wire/android/ui/userprofile/other/OtherUserProfileScreenViewModel.kt

```diff
@@ -59,11 +66,14 @@ class OtherUserProfileScreenViewModel @Inject constructor(
     private val userTypeMapper: UserTypeMapper,
     private val wireSessionImageLoader: WireSessionImageLoader,
     private val observeConversationRoleForUser: ObserveConversationRoleForUserUseCase,
-    qualifiedIdMapper: QualifiedIdMapper
+    qualifiedIdMapper: QualifiedIdMapper,
+    private val updateMemberRole: UpdateConversationMemberRoleUseCase
 ) : ViewModel() {
 
     var state: OtherUserProfileState by mutableStateOf(OtherUserProfileState())
-    var connectionOperationState: ConnectionOperationState? by mutableStateOf(null)
+
+    private val _infoMessage = MutableSharedFlow<UIText>()
+    val infoMessage = _infoMessage.asSharedFlow()
```

Comment:
```suggestion
    val infoMessage = infoMessage.asSharedFlow()
```

Reply:
But how? 😄  There is `private val _infoMessage` and `val infoMessage` - the first with the underscore is mutable one, but we don't want to share mutable one, that's why there is a second one - public non-mutable

Reply:
ohhhhh my bad i did read it as
```
    val _infoMessage = infoMessage.asSharedFlow()
```